src/meta/app/src/principal/user_privilege.rs

```diff
@@ -137,7 +154,7 @@ impl UserPrivilegeSet {
     /// on databases and tables, and has some Global only privileges.
     pub fn available_privileges_on_global() -> Self {
         let database_privs = Self::available_privileges_on_database();
-        let privs = make_bitflags!(UserPrivilegeType::{ Usage | Super | CreateUser | DropUser | CreateRole | DropRole | Grant | CreateDataMask });
+        let privs = make_bitflags!(UserPrivilegeType::{ Usage | Super | CreateUser | DropUser | CreateRole | DropRole | Grant | CreateDataMask | UsageExternalStage | ReadInternalStage | WriteInternalStage | UsageUDF });
```

Comment:
why gloabl？

Reply:
I think they are like (Select | Insert | Update ) which is used on a object thus not gloal?

Reply:
Sry, I forget to remove them from this list.

Reply:
Fixed.